README.md

```diff
@@ -52,6 +52,27 @@ $ dep ensure github.com/pkg/errors@^0.8.0
 
 See the help text for more detailed usage instructions.
 
+## Semantic Versioning
+
+`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The basic comparison operators are:
+
+* `=`: equal (aliased to no operator)
+* `!=`: not equal
+* `>`: greater than
+* `<`: less than
+* `>=`: greater than or equal to
+* `<=`: less than or equal to
+
```

Comment:
My personal favorite is the wildcard operator, because it's so readable. The wildcard character can be an upper- or lower-case `x`, or an astrix `*`. It lets you constraint to minor releases (`2.*`) or patch releases (`2.1.*`).

I find it easier than the sigils (`^2.1`), or the long-form ranges (`>=2.1, <3.0.0`)